share/kinesis-to-share/README.md

```diff
@@ -0,0 +1,209 @@
+This example will demonstrate how to publish data from [Amazon Kinesis](https://aws.amazon.com/kinesis/) to a [Vendia Share Uni](https://vendia.net/docs/share/dev-and-use-unis).  The point is to illustrate that partners to a Uni can take advantage of existing services they may already use, like Amazon Kinesis, to share information with partners.  In our scenario, a **Consignee** is placing an order for goods that should be published to a Uni that is comprised of a **Consignee**, **Shipper**, and **Carrier**.  In our scenario, the **Consignee** uses an order system that integrates with Amazon Kinesis.
+
+We will deploy the example using the [Vendia Share Command Line Interface (CLI)](https://vendia.net/docs/share/cli) and the [AWS Serverless Application Model (SAM)](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/what-is-sam.html).  Serverless resources like a [Kinesis](https://aws.amazon.com/kinesis/) stream and [AWS Lambda](https://aws.amazon.com/lambda/) function will be deployed.  Data will be published to an Kinesis stream via [generator.py](./generator.py), a Python3 script.  This, in turn, will trigger a Lambda function to parse order data and publish it to our node in the Vendia Share Uni.
+
+![kinesis-to-share Architecture](img/kinesis-to-share.png)
+
+# Pre-requisites
+
+* [Python3](https://www.python.org/download)
+
+* [AWS Serverless Application Model CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+
+* [AWS CLI version 2](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+
+* [Docker](https://docs.docker.com/install/)
+
+* [Vendia Share CLI](https://vendia.net/docs/share/cli)
+
+## Installing Python3 Dependencies
+
+```bash
+python3 -m venv venv
+. ./venv/bin/activate
+pip install pip --upgrade
+pip install -r requirements.txt
+```
+
+## Clone the Repository
+
+In order to use this example, you'll first need to clone the respository.
+
+### Clone with SSH
+
+```bash
+git clone git@github.com:vendia/examples.git
+```
+
+### Clone with HTTPS
+
+```bash
+git clone https://github.com/vendia/examples.git
+```
+
+### Change to the kinesis-to-share Directory
+
+```bash
+cd examples/share/kinesis-to-share
+```
+
+# Deploying the Example Uni
+
+This example will create a Uni to store attributes regarding shipments.
+
+If not already logged in to the share service do so by running [`share login`](https://vendia.net/docs/share/cli/commands/login):
+
+```bash
+share login
+```
+
+The `share uni create` command can be used to deploy our Uni.  You will need to copy the file `registration.json.sample` to `registration.json`.  Pick a unique `name` for your uni that begins with `test-` - by default all Unis share a common namespace so here is your chance to get creative.  Update the `userId` attribute of each node to reflect your Vendia Share email address.
+
+```bash
+cd uni_configuration
+share uni create --config registration.json
```

Comment:
should point to uni_configuration/registration.json given the current path

Reply:
The `share` command should be run from the `uni_configuration` directory (previous line).